gradle/dependencies.gradle

```diff
@@ -33,6 +34,10 @@ ext {
             // play services
             gmsLocation            : "com.google.android.gms:play-services-location:${version.gmsLocation}",
 
+            //architecture components
+            archLifecycleExtensions: "android.arch.lifecycle:extensions:${version.archLifecycleVersion}",
```

Comment:
We can save some weight here by using `android.arch.lifecycle:runtime:${version.archLifecycleVersion}` dependency instead of `extensions` as we are not using `ViewModel` or `LiveData`.

Reply:
Capturing from a chat that it seems like `runtime` does not contain `ProcessLifecycleOwner` so we need to stick to `extensions`.

Reply:
@electrostat any good reason why @LukasPaczos suggestion to cut on weight was not addressed and PR still got merged?

Reply:
@andrlee  Yes, per the above comment we discussed that `ProcessLifecycleOwner' is not included in `runtime`, so we had to keep `extensions`. We tested and discussed over slack the issue and was just captures in the brief above comment.

Reply:
we could still exclude `livedata` and `viewmodel` right? We shouldn't be including unnecessary deps.